agent/bench-scripts/pbench-user-benchmark

```diff
@@ -355,17 +355,17 @@ for rdir in ${result_dirs}; do
 	ln -s "$(basename "${rdir}")" "$(dirname "${rdir}")/reference-result"
 done
 
+${script_path}/postprocess/user-benchmark-wrapper "${benchmark_run_dir}" "${total_duration}"
```

Comment:
In all the other `bench-scripts` we invoke `pbench-metadata-log`, then `pbench-collect-sysinfo`, and then `pbench-tool-meister-stop`.  This change corrects `pbench-user-benchmark` to do the same.

However, we used to run the `user-benchmark-wrapper` after `pbench-metadata-log` and then `pbench-tool-meister-stop`, immediately before `pbench-collect-sysinfo`.  Preserving that sequence was not possible, so we now run the `user-benchmark-wrapper` before we run the finalization sequence.

I don't think that will be a problem, but it seemed running it after everything would make the user wait longer for their optional script to run than it did before.  This way it runs as soon as possible.